src/utils/number.js

```diff
@@ -0,0 +1,13 @@
+var isNumericString = function(n) {
+  return !isNaN(parseFloat(n)) && isFinite(n);
+}
+
+var isIntegerString = function(n) {
+  var n = parseFloat(n);
+  return Math.floor(n) === n;
```

Comment:
Should not work. Math.floor returns an integer but n is a string. So, the second check will be false.